From b73b57e49147e1c502b1fb3051555a7fe4187d50 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 11 Apr 2024 16:08:41 -0700
Subject: [PATCH] ==== docs improvement for mm ====

Subject: Docs/mm: rename legacy docs to unsorted docs

The work for changing the memory mangement documentation outline has
started from 2022.  For that, old documentation has placed under new
chapter, "Legacy Documentations".  The plan is to move all old
documentations into new outline, while asking new documents to be added
on the new outline only.

However, it is unclear where on the new outline each document should be
placed for some cases.  Meanwhile, the name ("legacy") make readers feel
the documents under the chapter might be outdated.  Also, apparently new
documents are added to the new outline without the guideline, just at
the bottom of the list.

Rename the chapter to simply "Unsorted Documentations" with minor
fixups.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/MRCkmupO | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/MRCkmupO

diff --git a/damon_meta_changes/MRCkmupO b/damon_meta_changes/MRCkmupO
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

